features/ambient-light.yml

```diff
@@ -0,0 +1,11 @@
+name: Ambient light sensor
+description: The `AmbientLightSensor` interface returns the current light level or illuminance of the ambient light around the device.
```

Comment:
>  current light level or illuminance of the ambient light around the device

Are these two different things? The MDN docs only show the one property that I can see.

Reply:
Also, the sensor APIs are a bit weird, right? They receive sensor reading events, rather than returning data themselves?